cypress/integration/profile.feature

```diff
@@ -0,0 +1,84 @@
+      | DEF      | data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAABLUlEQVRYR2O8cfPOf4ZBDBhHHUhh7IyGIIUByDAagqMhSGkIUKp/NA2OhiClIUCp/qGZBjdv3cVw49YdrJ7383FnUFdVxpDDp0daSoIhONCb4dev3wzLVqxj+PT5C1azrS1NGawsTFHksIYgyLJnz18wREUEMfDy8MA1HDtxmuHo8dMMfLw8GHK49CDb9vnLF7ADpSQlGHy93YiKfZIcCDIR5PA167YwiIgIgUOFnY0dbNGgcSDIMbCQRI7uQeVAbFE1qBz489dPhrXrt4KjFhbNQ8KB2HK+hpoKPEPAQh5bLkZWh5x7SM4kIM2DPgQHfRoc1Ll4UJeDg6omoWVdTHFVR1QlSSdFQ7O5RafAIcqa0RAkKpjwKBoNwdEQpDQEKNU/mgZHQ5DSEKBUPwCSacAYlWZThQAAAABJRU5ErkJggg==                                                                                                                             |
+
+  @negative
+  Scenario Outline: Set out of scope initials to <initials>
+    When I set initials to "<initials>"
+    Then initials is set to "<result>"
+    Examples:
+      | initials | result                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+      | 1234     | data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAAB0UlEQVRYR+2VPUsDQRCGJ01SeGchKWJEFET8BQE/ei0SBaOF+APFQiNoUmilhVGIFqJgEmNpTBEsvGtyjTInc+xt9nbP7AVE9srdm513n3l3JtVsdb7gD38pI1CzOoagJkAwBA1BXQK68caDhqAuAd34/+HBgTeA45OaD2NnuwiZdCYExnFdODiswKfj+uuTtgX7e2WwLSv0H53z1u0F61ulDVhaXIgErSTIJp/J54YEdt97cFSpQjY75e/hh5fp9z9gt1yC/HTOXyNxjuMG4s9qF9Bsd0AmUiqQxGEC2/6hwRPEJCiSJUZxKG6zuO7H1W8bcHf/IBSNpOk/HmWkQFYcJr+8qgOusQKJyvzcLKwuF4KzRZbAi/DxGBC1TocpS0w/qg5ib04CZWTYso9EkEf9G4Gtl1c4rZ5LvRVV9tglHlUgWQM9K3rxJOz6phH52tnciZZY9FJljZpe8dpKIeThsQgkcXx7kQkkmo9Pz8K+ifuJEYzT00RiRe0ncYIqcTJfivpoogLj+AgT0stmp5HqYsoS0xgbeN5QdXA82daEP+ZE+xjAz2T+vEw6HZosIgvE9qDK7OPaNwJ1yRqChqAuAd1440FDUJeAbrzxoC7Bb2Srxhg/WPQwAAAAAElFTkSuQmCC                                                                 |
```

Comment:
If we have 3 characters allowed to type, what we are expecting to get after typing 4 characters?

Reply:
For 3 or more characters always first 3 are initials